README.md

```diff
@@ -35,6 +35,55 @@ For example, if the archive is called "My Awesome Archive of Fandom Awesomeness"
 
 Note that the process will create databases and tables in MySQL as well as files in the designated folder. All the databases will be prefixed with CODENAME.
 
+
+## Process
+
+1. Run 
```

Comment:
This section feels kind of redundant compared to the Quickstart, unless you feel repeating the instructions more fully is necessary?